tests/ci/test_controller.py

```diff
@@ -1136,103 +1142,6 @@ class SelectDropdownOptionModel(ActionModel):
 		selected_value = await page.evaluate("document.getElementById('test-dropdown').value")
 		assert selected_value == 'option2'  # Second Option has value "option2"
 
-	async def test_extract_content_action(self, controller, browser_session, base_url, http_server):
```

Comment:
this test has been useful in the past for catching bugs in that action, can we keep it / get it passing?